oci/model.py

```diff
@@ -310,3 +310,66 @@ def as_dict(self):
             'mediaType': self.mediaType,
             'schemaVersion': self.schemaVersion,
         }
+
+
+@dataclasses.dataclass
+class OciContentDescriptor:
```

Comment:
I think this should be merged w/ `OciImageManifestListEntry`. Considering there are many optional attributes in both cases, likely, one single class should do.